src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/helpers/HmrcAccountMenuHelper.scala.html

```diff
@@ -0,0 +1,33 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.HmrcAccountMenu
+@import uk.gov.hmrc.hmrcfrontend.config.AccountMenuConfig
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.accountmenu._
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.language.{Cy, En}
+
+@this(hmrcAccountMenu: HmrcAccountMenu, accountMenuConfig: AccountMenuConfig)
+
+@(signOutHref: String)(implicit messages: Messages, request: RequestHeader)
+
+@hmrcAccountMenu(AccountMenu(
+  accountHome = AccountHome(href = accountMenuConfig.accountHomeHref),
+  messages = AccountMessages(href = accountMenuConfig.messagesHref),
```

Comment:
@kyle-bowden I'm sorry I've just realised I've lost track of one question I had that we didn't answer. @Muhammad-Ali-Ahmad the account messages can have a count on it https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/scala/uk/gov/hmrc/hmrcfrontend/views/viewmodels/accountmenu/AccountMessages.scala#L24 but we don't know how to populate it, can we pass through a parameter for it from this helper for now?

Reply:
@kyle-bowden I think another thing that's just clicked here for me is that we also need to be able to toggle the "active" status of the links - could we possibly do this automatically by checking the current request path, @Muhammad-Ali-Ahmad , wiring that in automatically might need to be done in a future change with the message count, and for now just allow it to be set

Reply:
Both of these suggestions are now available as features to the AccountMenu.